packages/nimble-components/src/utilities/style/appearance.ts

```diff
@@ -20,3 +20,14 @@ export function appearanceBehavior<AppearanceType>(
         styles
     );
 }
+
+export function appearanceVariantBehavior<AppearanceVariantType>(
```

Comment:
This function may be unneeded if you can use a private CSS variable instead (see my comment on styles.ts)